.github/advanced-issue-labeler.yml

```diff
@@ -0,0 +1,76 @@
+# NOTE: This is a common file that is overwritten by realm/ci-actions sync service
+# and should only be modified in that repository.
+
+# syntax - https://github.com/redhat-plumbers-in-action/advanced-issue-labeler#policy
+# Below keys map from the option used in issue form dropdowns to issue labels
+# Limitation:
+# Currently it's not possible to use strings containing ,␣ in the dropdown menus in the issue forms.
+
+---
+
+policy:
+  - template: [bug.yml, feature.yml]
+    section:
+      - id: [frequency]
+        label:
+          - name: 'Frequency:Once'
+            keys: ['Once']
+          - name: 'Frequency:Sometimes'
+            keys: ['Sometimes']
+          - name: 'Frequency:Always'
+            keys: ['Always']
+
+      - id: [repro]
+        label:
+          - name: 'Repro:Always'
+            keys: ['Always']
+          - name: 'Repro:Sometimes'
+            keys: ['Sometimes']
+          - name: 'Repro:No'
+            keys: ['No']
+
+      - id: [sync, flavour, services]
```

Comment:
it would also be helpful to have them include the OS, compiler etc.

Reply:
This is captured in the bug template: https://github.com/realm/realm-cpp/issues/new?assignees=&labels=T-Bug&projects=&template=bug.yml

I don't think it'll be a great idea to apply a label based on the OS/compiler since those are dynamic values, which means they'll generate many labels. Labels are generally useful when we want to run automations and/or aggregation - e.g. what percentage of filed tickets are for the local database vs for sync.

Reply:
Ahh, I see, since its present in the issue template we can resolve.